zipkin-lens/src/components/App/SidebarMenuItem.test.js

```diff
@@ -11,128 +11,98 @@
-          {...commonProps}
-          location={{ pathname: '/zipkin/somePage' }}
-        />,
-      );
-      expect(wrapper.find(ListItem).props().style).toEqual({
-        color: theme.palette.common.white,
-        backgroundColor: theme.palette.primary.dark,
-      });
-    });
-
-    it('should be darker when the current location does not equal to the first url', () => {
-      const wrapper = shallow(
-        <SidebarMenuItem.WrappedComponent
-          {...commonProps}
-          location={{ pathname: '/zipkin/otherPage' }}
-        />,
-      );
-      expect(wrapper.find(ListItem).props().style).toBeNull();
-    });
-
-    it('should push when clicked', () => {
-      const wrapper = shallow(
-        <SidebarMenuItem.WrappedComponent
-          {...commonProps}
-          location={{ pathname: '/zipkin/somePage' }}
-        />,
-      );
-      wrapper.find(ListItem).simulate('click');
-      expect(pushMock.mock.calls.length).toBe(1);
+    it('should render Tooltip', () => {
+      const wrapper = shallow(<SidebarMenuItemImpl {...props} />);
+      const tooltip = wrapper.find('[data-testid="tooltip"]');
+      expect(tooltip.length).toBe(1);
+      expect(tooltip.props().title).toBe('Internal Link');
     });
-  });
-
-  describe('should render external links', () => {
-    let pushMock;
-    let commonProps;
 
-    beforeEach(() => {
-      pushMock = jest.fn();
-
-      commonProps = {
-        history: { push: pushMock },
-        location: { pathname: '/zipkin/somePage' },
-        isExternalLink: true,
-        title: 'Some Page',
-        urls: ['https://example.com'],
-        buttonClassName: 'fas fa-search',
-      };
+    it('should render ListItem', () => {
+      const wrapper = shallow(<SidebarMenuItemImpl {...props} />);
+      const listItem = wrapper.find('[data-testid="list-item"]');
+      expect(listItem.length).toBe(1);
+      expect(listItem.props().button).toBe(true);
+      expect(listItem.props().onClick).toBeTruthy();
     });
 
-    it('should pass some props', () => {
-      const wrapper = shallow(<SidebarMenuItem.WrappedComponent {...commonProps} />);
-      const items = wrapper.find(ListItem);
-      expect(items.props().component).toBe('a');
-      expect(items.props().href).toBe('https://example.com');
+    it('should render Logo', () => {
+      const wrapper = shallow(<SidebarMenuItemImpl {...props} />);
+      const logo = wrapper.find('[data-testid="logo"]');
+      expect(logo.length).toBe(1);
+      expect(logo.props().component).toBe('span');
+      expect(logo.props().className).toBe('fab fa-home');
     });
 
-    it('should not call push when clicked', () => {
-      const wrapper = shallow(<SidebarMenuItem.WrappedComponent {...commonProps} />);
-      wrapper.find(ListItem).simulate('click');
-      expect(pushMock.mock.calls.length).toBe(0);
+    it('should call history.push when ListItem is clicked', () => {
+      const { getByTestId } = render(<SidebarMenuItemImpl {...props} />);
```

Comment:
add comment why history.push is important (I presume so you can hit back button)